pkgs/development/libraries/LAStools/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, cmake }:
+stdenv.mkDerivation rec {
+
+  version = "201003";
+  name = "LAStools-${version}";
```

Comment:
```suggestion
  pname = "LAStools;
  version = "201003";
```

Reply:
fixed in f93039565861622378b2de599df471bfb6db8d62